### PR TITLE
refactor: Standarized all the assignables previews to look the same

### PR DESCRIPTION
**Pull request by fermarinsanchez**
Refactor and redesign previews of tasks, tests and surveys

### CODE REVIEW

File: plugins/leemons-plugin-feedback/frontend/src/pages/private/feedback/StudentInstance/components/QuestionsCard.js [hunk 1]
```diff
@@ -70,97 +72,74 @@ function QuestionsCard({
     values: defaultValues || {},
   });
 
-  const moduleId = instance?.metadata?.module?.id;
-  const isModule = !!moduleId;
-  const moduleDashboardUrl = `/private/learning-paths/modules/dashboard/${moduleId}`;
+  // const moduleId = instance?.metadata?.module?.id;
+  // const moduleDashboardUrl = `/private/learning-paths/modules/dashboard/${moduleId}`;
```
Comment (MIGUELez11) [inline plugins/leemons-plugin-feedback/frontend/src/pages/private/feedback/StudentInstance/components/QuestionsCard.js:76]: Not used :D

```suggestion
```

File: plugins/leemons-plugin-feedback/frontend/src/pages/private/feedback/Preview/index_old.js
Comment (MIGUELez11) [inline plugins/leemons-plugin-feedback/frontend/src/pages/private/feedback/Preview/index_old.js:1]: File not used, maybe delete?

File: plugins/leemons-plugin-tests/frontend/src/pages/private/tests/Detail.js
Comment (MIGUELez11) [inline plugins/leemons-plugin-tests/frontend/src/pages/private/tests/Detail.js:1]: I would remove the commented code

**Review by MIGUELez11 — CHANGES_REQUESTED** [review]

File: plugins/leemons-plugin-feedback/frontend/src/pages/private/feedback/StudentInstance/components/QuestionsCard.js (hunk 1 above)
Comment (fermarinsanchez) [reply plugins/leemons-plugin-feedback/frontend/src/pages/private/feedback/StudentInstance/components/QuestionsCard.js:76]: Ok. done!

File: plugins/leemons-plugin-feedback/frontend/src/pages/private/feedback/Preview/index_old.js
Comment (fermarinsanchez) [reply plugins/leemons-plugin-feedback/frontend/src/pages/private/feedback/Preview/index_old.js:1]: Ok. done!

**Review by MIGUELez11 — APPROVED** [review]

**Review by MIGUELez11 — APPROVED** [review]

**Pull request merged**